superset/db_engine_specs/databricks.py

```diff
@@ -15,11 +15,34 @@
 # specific language governing permissions and limitations
 # under the License.o
 
+from datetime import datetime
+from typing import Optional
+
+from superset.db_engine_specs.base import BaseEngineSpec
 from superset.db_engine_specs.hive import HiveEngineSpec
 
 
 class DatabricksHiveEngineSpec(HiveEngineSpec):
     engine = "databricks"
-    engine_name = "Databricks"
+    engine_name = "Databricks (Hive)"
```

Comment:
nit: to avoid confusion, maybe this should be "Databricks (PyHive)"

Reply:
I talked with Eugenia and she suggested "Databricks Interactive Cluster" here, and "Databricks SQL Endpoint" for ODBC.